platform/security/filter/security-filter-csrf/src/main/java/org/codice/ddf/security/filter/csrf/CsrfFilter.java

```diff
@@ -290,17 +276,14 @@ private void respondForbidden(HttpServletResponse httpResponse, String msg) {
     }
   }
 
-  public void setUserAgentWhitelist(List<String> userAgentWhitelist) {
-    if (userAgentWhitelist == null) {
-      return;
-    }
-
-    this.userAgentWhitelist.clear();
-
-    for (String agent : userAgentWhitelist) {
-      Pattern userAgentPattern = Pattern.compile(agent);
-      this.userAgentWhitelist.add(userAgentPattern);
-    }
+  private boolean isNotBrowser(String userAgentHeader) {
+    return userAgentHeader == null
+        || !(userAgentHeader.contains("Mozilla")
+            || userAgentHeader.contains("Safari")
+            || userAgentHeader.contains("OPR")
+            || userAgentHeader.contains("MSIE")
+            || userAgentHeader.contains("Edge")
+            || userAgentHeader.contains("Chrome"));
```

Comment:
❓ Is there a reason you didn't just do the negated condition (here and in the if statement above) and name the method `isBrowser`? It's a little less readable with the double negation.

Reply:
I wanted to keep it similar to the ECP code